src/controllers/extension-limit-reached.controller.ts

```diff
@@ -0,0 +1,27 @@
+import {NextFunction, Request, Response} from "express";
+import * as sessionService from "../services/session.service";
+import * as templatePaths from "../model/template.paths";
+import {ExtensionsCompanyProfile, getCompanyProfile} from "../client/apiclient";
+import logger from "../logger";
+import * as errorMessages from "../model/error.messages";
+
+export const render = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+  const companyNumber: string = sessionService.getCompanyInContext(req.chSession);
+  if (companyNumber) {
+    try {
+      const token: string = req.chSession.accessToken() as string;
+      const company: ExtensionsCompanyProfile = await getCompanyProfile(companyNumber, token);
+      res.render(templatePaths.EXTENSION_LIMIT_REACHED, {
+        companyName: company.companyName,
```

Comment:
if the key is the same name as the value you can just use the key, ie companyName, companyNumber, monthLimit: 12,

Reply:
reduced this to just the values.

Reply:
sorry, that won't work unless you have the variable name the same ie var called companyName. It is ok as it was : company.companyName.  My fault, but handy to know in future.

Reply:
Just tested it the chnage seems fine.